husky_components/datagrid/decorators/table-view.js

```diff
@@ -1092,8 +1099,10 @@ define(function() {
         editableInputFocusoutHandler: function(event) {
             if (!!this.isFocusoutHandlerEnabled) {
                 this.sandbox.dom.stopPropagation(event);
-                var recordId = this.sandbox.dom.data(this.sandbox.dom.parents(event.currentTarget, '.' + constants.rowClass), 'id');
-                this.editRow(recordId);
+
+                var e = jQuery.Event('keypress');
```

Comment:
Imo a better approach would be to create a states list for all the inputs and use the input state to check if you need to save the entry.